node: ci fails for lack of "caches" in "knownGlobals"

After Deno canary version b3122279e58e51520a38e51cca317a09cdadd7cb4, most tests run in `deno task node/test` fail with the following error:


```mjs
Node.js compatibility "internet/test-dgram-connect.js" ...
------- output -------
error: Uncaught AssertionError: Unexpected global(s) found: caches
    Error.captureStackTrace(this, stackStartFn || stackStartFunction);
          ^
    at new AssertionError (file:///home/runner/work/deno_std/deno_std/node/assertion_error.ts:532:11)
    at createAssertionError (file:///home/runner/work/deno_std/deno_std/node/assert.ts:45:17)
    at FunctioError: "internet/test-dgram-connect.js" failed
n.fail (file:///home/runner/work/deno_std/deno_std/node/assert.ts:396:11)
    at process.<anonymous> (file:///home/runner/work/deno_std/deno_std/node/_tools/test/common/index.js:105:14)
    at process.emit (file:///home/runner/work/deno_std/deno_std/node/_events.mjs:391:35)
    at file:///home/runner/work/deno_std/deno_std/node/process.ts:323:15
    at innerInvokeEventListeners (deno:ext/web/02_event.js:755:9)
    at invokeEventListeners (deno:ext/web/02_event.js:795:7)
    at dispatch (deno:ext/web/02_event.js:664:11)
    at dispatchEvent (deno:ext/web/02_event.js:1043:14)
```

The reason seems to be that "caches" was added to global by https://github.com/denoland/deno/pull/15829, but it is not yet supported in `node/_tools/test/common/index.js`.

--- a/node/_tools/test/common/index.js
+++ b/node/_tools/test/common/index.js
@@ -24,6 +24,7 @@ let localhostIPv4 = null;
 let knownGlobals = [
   atob,
   btoa,
+  caches,
   clearImmediate,
   global.clearInterval,
   global.clearTimeout,

